submissions/kovalov/dom/scripts/element.js

```diff
@@ -0,0 +1,11 @@
+export function createElement({ name, description, src }) {
+  const HTML = `
+	 <div>
+			 <h1 class="main__title">${name}</h1>
+			 <img class="main__image" src="./assets/images/${src}" alt="Photo of ${name}.">
+			 <p class="main__description">${description}</p>
+	 </div>`;
+  const template = document.createElement('template');
+  template.innerHTML = HTML;
+  return template.content.firstElementChild;
```

Comment:
Why are you using Template Element? It is makes sense if `<template>` existing into your html and you get something from there to push into html.  But you don't do this. It is better create `<div>` or `fragment` and return it.
It looks realy strenge.

Reply:
Have used template to prevent creation of a few elements.

Reply:
You can create `DIV` and pass your template string with data item values as `innerHTML` and return  this `DIV` from your element creator.